Resources/Prototypes/Entities/Objects/Fun/crayons.yml

```diff
@@ -88,13 +88,21 @@
   - type: Crayon
     color: Red
     selectableColor: true
-    capacity: 30
+    capacity: 1000000 # Delta V - Let there be art!
   - type: Tag
     tags:
     - Write
     - Crayon
     - Recyclable
     - Trash
+  - type: SolutionContainerManager # Delta V - So they can't accidentally eat it
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 99998
```

Comment:
This value seems absurd. Even if the intent is to disallow them eating it, making it the greatest source of food in the game is not that. Just disable eating crayons if that's the idea, or make it an option that can only be done via verb menu.